README.md

```diff
@@ -1,79 +1,196 @@
+## Timelines
 
-**Q:** How do I send a direct message?
+### Why is the count of likes and boosts different to the web?
 
-**A:** Create a toot, mention the user and set the visibility (World icon) to Direct.
-##
+This is because of how Mastodon works. Likes and boosts do not federate perfectly; if you and the other user are on different servers you will see different numbers between Tusky and the website.
 
-Template:
-**Q:**
+### How do I see the Federated timeline?
 
-**A:**
-##
+From your home timeline:
+
+- Open "Account Preferences" (swipe from the left edge, or tap your profile picture to see the option)
+- Tap "Tabs"
+- Tap the "+" button (bottom right)
+- Choose "Federated" from the menu
+
+You can adjust the order of tabs on this screen too.
+
+### How do I hide boosted posts from my timeline?
+
+- Open Preferences
+- Tap the "Filters > Tabs" section
+- Unselect the "Boosts" option
+
+### How do I hide replies to posts from my timeline?
+
+- Open Preferences
+- Tap the "Filters > Tabs" section
+- Unselect the "Replies" option
+
+### How do I follow a hashtag?
+
+- Tap the hashtag in a post to view all posts with this hashtag
+- Tap the "follow" icon at the top right
+
+### How do I mute a hashtag?
+
+> This feature is available in Tusky 21 or above (including Nightly). To check the Tusky version you are using open the "About" menu.
+
+- Tap the hashtag in a post to view all posts with this hashtag
+- Tap the "mute" icon at the top right
+
+## Posting
+
+### How do I edit a post?
+
+> This feature is only available in Tusky 21 or above (including Nightly). To check the Tusky version you are using open the "About" menu.
+
+Find the post you want to edit, then:
+
+- Tap the "..." menu at the bottom-right of the post
+- Choose "Edit"
+- Tap "Toot" to send the edited post (replacing the original)
+
+### Why does Tusky only allow 500 characters when my instance has a custom character limit?
+
+Your instance is not configured to report that it supports a custom limit using the Mastodon API.
+
+This is something that your instance admins will need to change for you, Tusky cannot change it.
+
+You can contact them and say:
+
+> The instance is not exporting the `max_toot_chars` property at the `/api/v1/instance` endpoint. See e.g. here https://pleroma.site/api/v1/instance for a config that works with Tusky.
+
+### How do I send a direct message?
+
+> Important: Direct messages on Mastodon are not encrypted, and can be read by the admins of your instance. Be careful sharing any private information on Mastodon.
+
+Create a new post, @-mention the user you want to send it to, then tap the "visibilty" icon (by default this looks like a globe) and set the post's visibility to "Direct".
+
+## Notifications
+
+### Why are notifications less frequent with Tusky?
+
+By default, Tusky checks for new notifications approximately every 15 minutes.
+
+Tusky can also use [Unified Push](https://unifiedpush.org/) to show notifications in real-time. To do this:
```

Comment:
Didn't feel like information that usefully solves the user's problem.

They've come with a question -- why am I not getting notifications as fast as I expect? Knowing why Tusky uses UP doesn't help solve their actual problem.

And I don't think the previous justification ("avoid proprietary Google libraries") is actually correct. com.google.android.gms is Apache 2.0 licensed (https://mvnrepository.com/artifact/com.google.gms/google-services/4.2.0), as is com.google.firebase:firebase-messaging (https://mvnrepository.com/artifact/com.google.firebase/firebase-messaging/23.1.1).

There are reasons not to use Firebase (e.g., vendor lock-in, not wanting user data to go through a third party), but I don't think this FAQ is the best place for it. As part of the privacy policy, maybe. Or a design decisions document.

Reply:
…but this is a question that people ask, "Why bother with polling/UP instead of standard google services"

Reply:
Q: Why does Tusky use UnifiedPush and does not work with Google Services out of the box?
A: Because Google Services don't work on all devices and because we want to give users the freedom to chose their push provider.

Reply:
Done.